app.go

```diff
@@ -438,3 +449,81 @@ func executeCancelSell(cmd command) bool {
 
 	return true
 }
+
+func executeSetBuyAmount(cmd command) bool{
+	account := accountStore.GetAccount(cmd.UserID)
+
+	if account == nil {
+		log.Infof("User %s does not have an account", cmd.UserID)
+		return false
+	}
+
+	amount, err := currency.NewFromString(cmd.Args[1])
```

Comment:
cmd.Args should be put into local variables because: 

1) You'll probably use them again
2) We don't know what they refer to off hand.